x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/operator/DriverContext.java

```diff
@@ -51,6 +52,13 @@ public BigArrays bigArrays() {
         return bigArrays;
     }
 
+    /**
+     * The {@link CircuitBreaker} to use to track memory.
+     */
+    public CircuitBreaker breaker() {
+        return bigArrays.breakerService().getBreaker(CircuitBreaker.REQUEST);
```

Comment:
I know that you need this here, but that can also come in the other PR that adds the BlockFactory.